### PR TITLE
Fixing stats for Android

### PR DESCRIPTION
**Pull request by filipi87**
Fixing stats for Android

### CODE REVIEW

**Review by mattieruth — APPROVED** [review]
common sense tells me this change is good 👍

File: android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java [hunk 1]
```diff
@@ -254,10 +254,10 @@ public void receiverGetStats(String receiverId, Promise promise) {
     }
 
     public void senderGetStats(String senderId, Promise promise) {
-        RtpReceiver targetSender = null;
-        for (RtpReceiver r : peerConnection.getReceivers()) {
-            if (r.id().equals(senderId)) {
-                targetSender = r;
+        RtpSender targetSender = null;
+        for (RtpSender s : this.peerConnection.getSenders()) {
```
Comment (mattieruth) [inline android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java:258]: one question: here you added `this.` -- Do we need to change that in `receiverGetStats()` too? or was the addition of `this` somehow not necessary?

Comment (filipi87) [reply android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java:258]: `this` keyword is used to refer to the current instance of the class in which it appears. But it is not mandatory, it is just to make it more explicit.

**Comment by filipi87** [comment]
Thank you for the review @mattieruth 🎉

**Pull request merged**